runtime/wasm/src/module/mod.rs

```diff
@@ -953,6 +955,30 @@ impl WasmiModule {
         self.ctx.host_exports.log_log(&self.ctx.logger, level, msg);
         Ok(None)
     }
+
+    /// function transactionData(txId: string): Bytes | null
```

Comment:
Should we add `arweave.` and `box.`  prefixes to these comments?

Reply:
Done.